### PR TITLE
Added moving to the starting angle of series of measuremets,

### PR DESCRIPTION
**Pull request by MatejKomlossy**
changes in setNumberOfScansToAverage in Settings

### CODE REVIEW

File: project/src/measurement/SeriesOfMeasurements.java [hunk 1]
```diff
@@ -13,9 +23,114 @@
...
+                stepperMotor.moveOnePulseBackwards(currentAngleLabel);
+            }));
+        }
+        moving.setCycleCount(stepperMotor.pulsesNeededToMove(angle));
+        moving.setOnFinished(e -> startSeries(chart, currentAngleLabel, remainingStepsLabel));
+        moving.play();
+    }
+
+    private void startSeries(Chart chart, Label currentAngleLabel, Label remainingStepsLabel) {
+        Double interval = (Settings.getIntegrationTime()/1000) + chart.getDrawingTime() + stepperMotor.getStepTime();
```
Comment (FilipSekerka) [inline project/src/measurement/SeriesOfMeasurements.java]: netreba aj tuto riesit ten average mod? ze sa vynasobi ten interval

File: project/src/measurement/SeriesOfMeasurements.java [hunk 2]
```diff
@@ -13,9 +23,114 @@
...
+        }
+    }
+
+    private void substractBackgroundIfNeeded(double[] values){
+        double[] background = Settings.getBackground();
+        if(Settings.getSubtractBackground() == false || background == null){
+            return;
+        }
+        for (int i = 0; i < Math.min(values.length, background.length); i++) {
+            values[i] = Math.max(values[i] - background[i], 0);
```
Comment (FilipSekerka) [inline project/src/measurement/SeriesOfMeasurements.java:127]: nechceme tu nechat tu zapornu hodnotu?

**Review by FilipSekerka — COMMENTED** [review]
pozri tie 2 veci prosim ta

File: project/src/measurement/SeriesOfMeasurements.java (hunk 1 above)
Comment (MatejKomlossy) [reply project/src/measurement/SeriesOfMeasurements.java]: pravda

**Review by FilipSekerka — APPROVED** [review]

**Pull request merged**